CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 - Add Azure Pipelines Scaler ([#1706](https://github.com/kedacore/keda/pull/1706))
 - Add OpenStack Metrics Scaler ([#1382](https://github.com/kedacore/keda/issues/1382))
 - Added basic, tls and bearer authentication support to the Prometheus scaler [#1559](https://github.com/kedacore/keda/issues/1559)
+- Add header Origin to apache artemis scaler
```

Comment:
```suggestion
- Add header Origin to Apache Artemis scaler [#1796](https://github.com/kedacore/keda/pull/1796)
```

Reply:
https://issues.apache.org/jira/browse/ARTEMIS-2779
I find it in this issue, I'll try with 2.11 and resolve the conversation.

Reply:
tested on 2.11.0, it works